tests/depth-dependent-box-file-simple/depth_average.gnuplot

```diff
@@ -4,144 +4,144 @@
 #
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
-# <x> <y> <temperature> <adiabatic_temperature> <velocity_magnitude> <sinking_velocity> <Vs> <Vp> <viscosity> 
-0 -1e+07 398 0 7.35903e-32 3.16065e-09 -1 -1 1.6e+21 
-300000 -1e+07 398 0 7.35903e-32 3.16065e-09 -1 -1 1.6e+21 
+# <x> <y> <temperature> <adiabatic_temperature> <velocity_magnitude> <sinking_velocity> <Vs> <Vp> <viscosity> <vertical_heat_flux> 
+0 -1e+07 398 0 7.35903e-32 6.30385e-10 -1 -1 1.6e+21 0.00333333 
+300000 -1e+07 398 0 7.35903e-32 6.30385e-10 -1 -1 1.6e+21 0.00333333 
```

Comment:
Something is strange here -- the velocity is around 7e-32 but the sinking velocity is around 6e-9. This makes no real sense.